lambda/src/main/scala/pricemigrationengine/handlers/NotificationEmailHandler.scala

```diff
@@ -11,27 +12,71 @@ import zio.{Runtime, ZEnv, ZIO, ZLayer, clock, console}
 object NotificationEmailHandler {
   private val NotificationEmailLeadTimeDays = 30
```

Comment:
Perhaps should go in config

Reply:
I thought about that but you need to make a code change to change the config version in the cloudformation template to get any change to the config to take effect. 

So it seems easier to just have the config in code. Its a drawbacks of using cloudformation dynamic references for config.

Reply:
👍